ui/pages/confirmations/components/confirm/info/approve/approve.tsx

```diff
@@ -98,6 +192,7 @@ const ApproveInfo = () => {
     <>
       <ApproveStaticSimulation />
       <ApproveDetails />
+      {!isNFT && <SpendingCap />}
```

Comment:
If the core addition here is the spending cap, is it worth updating the PR title and description for clarity?